src/gosling-track/gosling-track.ts

```diff
@@ -480,7 +491,7 @@ const factory: PluginTrackFactory<Tile, GoslingTrackOptions> = (HGC, context, op
 
             const flatTileData = ([] as Datum[]).concat(...models.map(d => d.data()));
             if (flatTileData.length !== 0) {
-                publish('rawData', { id: context.viewUid, data: flatTileData });
+                this.options.siblingIds.forEach(id => publish('rawData', { id, data: flatTileData }));
```

Comment:
This basically means that the rawData event is fired multiple times with the same data for different track Ids?

Reply:
This seems to be resolved!